randovania/games/am2r/json_data/Hydro Station.txt

```diff
@@ -103,7 +103,16 @@ Extra - minimap_data: [{'x': 29, 'y': 18}, {'x': 29, 'y': 19}, {'x': 29, 'y': 20
           Speed Booster and Shinesparking Tricks (Beginner)
           # IBJ
           Infinite Bomb Jumping (Advanced) and Can Use Bombs
-          Morph Ball and Morph Glide (Beginner)
+          All of the following:
+              # Morph Glide
+              Morph Ball
+              Any of the following:
+                  All of the following:
+                      # Without HJ
+                      Morph Glide (Expert)
+                      Power Grip or Walljump (Advanced)
+                  # With HJ
+                  Hi-Jump Boots and Morph Glide (Advanced)
```

Comment:
The glide requirements dont seem right here at all. 
Without PG or WJ, its an expert
with PG or WJ its intermediate
with HJ its beginner

Reply:
I have to say the `No HJ + PG or WJ` feels greater than intermediate due to the distance. See the below image for an intermediate Morph Glide we have already.
![image](https://github.com/randovania/randovania/assets/125271738/7ffb2d3c-056e-4a54-9153-4c25756e83a0)


Agree with the rest.

Reply:
The one from a3 is actually a bit tigher if i remember correctly
The one in a2 is *very* easy, as you can pretty much just spam morph